src/network_analysis.jl

```diff
@@ -708,13 +792,13 @@ Compute conserved quantities for a system with the given conservation laws.
 """
 conservedquantities(state, cons_laws) = cons_laws * state
 
-# If u0s are not given while conservation laws are present, throws an error.
+# If u0s are not given while conservation laws are present, throw an error.
 # Used in HomotopyContinuation and BifurcationKit extensions.
-# Currently only checks if any u0s are given
-# (not whether these are enough for computing conserved quantitites, this will yield a less informative error).
+# Currently only checks if any u0s are given (not whether these are enough for computing
+# conserved quantities, this will yield a less informative error).
 function conservationlaw_errorcheck(rs, pre_varmap)
     vars_with_vals = Set(p[1] for p in pre_varmap)
-    any(s -> s in vars_with_vals, species(rs)) && return
+    any(sp -> sp in vars_with_vals, species(rs)) && return
```

Comment:
We don't really use `s` as much throughout, changed this to `sp` which is more commonly used (and also more clearly a species).